lib/state/action-types.ts

```diff
@@ -67,6 +67,7 @@ export type ToggleSimperiumConnectionStatus = Action<
 >;
 export type ToggleEditMode = Action<'TOGGLE_EDIT_MODE'>;
 export type ToggleTagDrawer = Action<'TAG_DRAWER_TOGGLE', { show: boolean }>;
+export type ToggleTrash = Action<'TRASH_TOGGLE'>;
```

Comment:
This doesn't matter really, but it's odd that everywhere else we use `toggle` and then `trash` but here we use for the action name `trash` then `toggle`.

Calypso has some standards setup for organization that push the noun/receiver first but that was also in part because of a lack of a type system to prevent overlap of names. Here we have fewer actions plus we have a type system so if there's a chance we feel obliged to hold Calypso style I wouldn't worry about it.

Reply:
I deleted this code in the next commit.  Yes, we should have a standard.